Makefile.in

```diff
@@ -54,9 +57,11 @@ configure config.h.in: configure.ac
 	./autogen.sh
 
 install: all
+	$(MKDIR) -p $(DESTDIR)$(bindir)
 	$(MKDIR) -p $(DESTDIR)$(sbindir)
 	$(MKDIR) -p $(DESTDIR)$(mandir)/man1
-	$(INSTALL) -m 0755 $(PROG) $(DESTDIR)$(sbindir)
+	$(INSTALL) -m 0755 $(PROG) $(DESTDIR)$(bindir)
+	$(LN_S) $(DESTDIR)$(bindir)/$(PROG) $(DESTDIR)$(sbindir)/$(PROG)
```

Comment:
Oh yikes, we've opened a can of worms, it seems. ;-)
The command works now, but `$(DESTDIR)` should not appear in the symlink. The easy fix:

```
$(LN_S) $(bindir)/$(PROG) $(DESTDIR)$(sbindir)/$(PROG)
```

leaves a dangling symlink in the installation path for a non-empty `$(DESTDIR)`. Not fatal, but ugly.
Using a relative symlink seems preferrable, but requires external tools like eg.

```
realpath --relative-to=$(DESTDIR)$(bindir) $(DESTDIR)$(sbindir)/$(PROG)
```

to determine the relative path. I guess this won't be available on your systems as a buld-time dependency?

Reply:
I don't understand the statement about `$(DESTDIR)`. If it is empty, it will be like your statement, if not it still will be consistent.
I think you are likely talking about the fact that the build is staged in `$(DESTDIR)`, but then moved to final `prefix`?

DARN. You are right, I can reproduce this:
```
osipovmi@deblndw011x:/var/tmp/stage
$ tree
.
└── tmp
    └── msktutil
        ├── bin
        │   └── msktutil
        ├── sbin
        │   └── msktutil -> /var/tmp/stage/tmp/msktutil/bin/msktutil
        └── share
            └── man
                └── man1
                    └── msktutil.1

7 directories, 3 files
```
moved
```
osipovmi@deblndw011x:/tmp
$ tree msktutil/
msktutil/
├── bin
│   └── msktutil
├── sbin
│   └── msktutil -> /var/tmp/stage/tmp/msktutil/bin/msktutil
└── share
    └── man
        └── man1
            └── msktutil.1

5 directories, 3 files
```

Reply:
If I understand [this resource](https://stackoverflow.com/a/11307770/696632) correctly, you are right about:
```
$(LN_S) $(bindir)/$(PROG) $(DESTDIR)$(sbindir)/$(PROG)
```

Let me check that...

Reply:
I think this is it:
```patch
$ git diff
diff --git a/Makefile.in b/Makefile.in
index a3f94e9..838de35 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -61,7 +61,7 @@ install: all
     $(MKDIR) -p $(DESTDIR)$(sbindir)
     $(MKDIR) -p $(DESTDIR)$(mandir)/man1
     $(INSTALL) -m 0755 $(PROG) $(DESTDIR)$(bindir)
-    $(LN_S) $(DESTDIR)$(bindir)/$(PROG) $(DESTDIR)$(sbindir)/$(PROG)
+    $(LN_S) -f $(bindir)/$(PROG) $(DESTDIR)$(sbindir)/$(PROG)
     $(INSTALL) -m 0644 $(MAN) $(DESTDIR)$(mandir)/man1
```

Reply:
Used:
```
$ ./configure --with-krb5=mit --prefix=/tmp/msktutil
...
$ make DESTDIR=/var/tmp/stage install
```

Reply:
With
```
$ tree /var/tmp/stage
/var/tmp/stage
└── tmp
    └── msktutil
        ├── bin
        │   └── msktutil
        ├── sbin
        │   └── msktutil -> /tmp/msktutil/bin/msktutil
        └── share
            └── man
                └── man1
                    └── msktutil.1
```
then
```
$ tree /tmp/msktutil/
/tmp/msktutil/
├── bin
│   └── msktutil
├── sbin
│   └── msktutil -> /tmp/msktutil/bin/msktutil
└── share
    └── man
        └── man1
            └── msktutil.1

5 directories, 3 files
```

WDYT?